### PR TITLE
Expose ImageLoader comp from Viewer

### PR DESCRIPTION
**Pull request by JF-Cozy**
Pour un besoin MesPapiers, ce composant est nécessaire en tant que tel

BC : j'ai quand même déclencher un BC pour indiquer que ImageLoader n'est plus accessible depuis le dossier `/Viewer`

### CODE REVIEW

**Comment by Merkur39** [comment]
C'est tout bon pour moi 👍

**Comment by JF-Cozy** [comment]
il faut une main innocente pour valider  @Ldoppea :)

File: react/ImageLoader/index.spec.jsx [hunk 1]
```diff
@@ -116,4 +123,31 @@ describe('ImageLoader', () => {
...
+    const { root } = setup({ file })
+    const { getByTestId } = root
+    getMock.mockResolvedValue({
+      data: {
+        links: {
+          icon: 'http://urlnotvalidneither'
+        }
+      }
+    })
+    checkImageSource.mockRejectedValueOnce('KO')
```
Comment (Ldoppea) [inline react/ImageLoader/index.spec.jsx:147]: What is the goal of this 2nd mock?

**Review by Ldoppea — COMMENTED** [review]
If the goal of `ImageLoader` is to load an image from a `file`, then maybe we should name it `FileImageLoader`? (or similar)

This name would be more descriptive on what it does. `ImageLoader` is too generic in my opinion.

As we are making a BreakingChange I suppose it is a good time to improve the name?

Everything else looks good to me.

File: react/ImageLoader/index.spec.jsx (hunk 1 above)
Comment (Merkur39) [reply react/ImageLoader/index.spec.jsx:147]: No goal, just a mistake, it's fixed 👍

**Review by Ldoppea — APPROVED** [review]

**Pull request merged**